src/main/scala/com/databricks/spark/redshift/RedshiftJDBCWrapper.scala

```diff
@@ -222,7 +222,17 @@ private[redshift] class JDBCWrapper {
     }.getOrElse {
       throw new IllegalArgumentException(s"Did not find registered driver with class $driverClass")
     }
-    driver.connect(url, new Properties())
+    val properties = new Properties()
+    credentials match {
+      case Some((user, password)) => {
+        if (url.contains("user=") || url.contains("password=")) {
+          throw new IllegalArgumentException(s"Ambiguous Credentials: User and password must be passed as options or within the URL, not both")
```

Comment:
This line is too long; please break up this string so that no line exceeds 100 characters.